compiler/rustc_interface/src/tests.rs

```diff
@@ -558,6 +558,7 @@ fn test_debugging_options_tracking_hash() {
     tracked!(link_only, true);
     tracked!(merge_functions, Some(MergeFunctions::Disabled));
     tracked!(mir_emit_retag, true);
+    tracked!(mir_opt_skip_pass, vec!["ConstProp".to_owned()]);
```

Comment:
I think you want 
```suggestion
    tracked!(mir_opt_skip_pass, vec![]);
```

Reply:
I definitely do not want this. See the comment above this block.